pkgs/by-name/wt/wtf/package.nix

```
@@ -0,0 +1,34 @@
+{ pkgs ? import <nixpkgs> { }, lib ? pkgs.lib, fetchFromGitHub, }:
+
+pkgs.stdenv.mkDerivation rec {
```

We usually dont go with `pkgs` directly, but import what the derivation needs. Lets also use `finalAttrs` for better override flexibility (see #315337)
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

Thanks for this tips.